res/card/template/passive.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>card creator</title>
+</head>
+
+<!-- Hack to get emojis to render -->
+<style>
+  img.emoji {
+     height: 1em;
+     width: 1em;
+     margin: 0 .05em 0 .1em;
+     vertical-align: -0.1em;
+  }
+</style>
+
+<!-- https://stackoverflow.com/questions/687998/auto-size-dynamic-text-to-fill-fixed-size-container -->
+<script src="https://twemoji.maxcdn.com/2/twemoji.min.js?11.2"></script>
+<script type="text/javascript">
+  window.onload=function shrink()
+  {
+    const rules_text = document.getElementByClassName('active')[0] 
+    rules_text.innerHTML = ":^)"
+
+    twemoji.parse(document.body);
+    textSpan = document.getElementById("title-text")
+    textDiv = document.getElementById("title")
+    
+    textSpan.style.fontSize = "36pt"
+
+    while(textSpan.offsetHeight > textDiv.offsetHeight)
+    {
+        updated_font_size = parseInt(textSpan.style.fontSize) - 1
+        textSpan.style.fontSize = updated_font_size + "pt";
+    }
+  }
+</script>
+
+
+<body>
+  <div class="card">
+      <div class="tier"></div>
+      <div class="border"></div>
+      <div class="frame"></div>
+      <div class="title" id="title">
+        <span class="title-text vertical-center" id="title-text">{{title}}</div>
+      </div>
+      <div class="illustration"></div>
+      <div class="type">
+        <div class="type-text">
+          {{upgrade}}
+        </div>
+      </div>
+      <div class="rules">
+        <div class="rules-text">
+          <div class="active">
+            <div class="cast glyph"></div> {{active_rules}}
+          </div>
+          <div class="passive">
+            <div class="hand glyph"></div> {{passive_rules}}
+          </div>
```

Comment:
Copied this file from the default template so that we could have different versions depending on the rules text.

There's a lot of shared content here, but I'm just going to leave in some duplication for now